rest-api/src/main/java/gov/cms/qpp/conversion/api/services/StorageServiceImpl.java

```diff
@@ -53,8 +54,9 @@
 		final String bucketName = environment.getProperty(Constants.BUCKET_NAME_ENV_VARIABLE);
 		final String kmsKey = environment.getProperty(Constants.KMS_KEY_ENV_VARIABLE);
 		if (Strings.isNullOrEmpty(bucketName) || Strings.isNullOrEmpty(kmsKey)) {
-			API_LOG.warn("No bucket name is specified or no KMS key specified.");
-			return CompletableFuture.completedFuture("");
+			String error = "No bucket name is specified or no KMS key specified.";
+			API_LOG.warn(error);
+			return FutureHelper.exceptionally(new IllegalStateException(error));
```

Comment:
We don't want to do this because that will prevent the audit information in DynamoDB from being written.  This would occur if the the S3 bucket is not specified, but the KMS key and DynamoDB table are specified.